debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/RecordMakers.java

```diff
@@ -61,12 +63,13 @@
      * @param topicSelector the selector for topic names; may not be null
      * @param recorder the potentially blocking consumer function to be called for each generated record; may not be null
      */
-    public RecordMakers(SourceInfo source, TopicSelector topicSelector, BlockingConsumer<SourceRecord> recorder) {
+    public RecordMakers(SourceInfo source, TopicSelector topicSelector, BlockingConsumer<SourceRecord> recorder, Configuration config) {
```

Comment:
I'd prefer to just inject the required option instead of the config.

Reply:
I included the whole config object because I thought it might be useful in the future? But I'm happy to change it, just to be a specific option

Reply:
Yes, I think it's better to have just the needed options here. It still can be generalized down the road, if actually needed.